### PR TITLE
Basic web app

### PR DESCRIPTION
**Pull request by RaulinN**
Basic web app project includes : 

- Correction of the "infinite-width" bug on web only
- Correction of #16 
- Correction of #17

### CODE REVIEW

**Review by imaximumGit — CHANGES_REQUESTED** [review]
The STRINGS object is import twice in the file _Organization.js_ and _Attendee.js_

**Comment by RaulinN** [comment]
Nice one @imaximumGit! I removed all unwanted imports in the above commit

**Review by imaximumGit — APPROVED** [review]

File: PopApp/Components/Attendee.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import {StyleSheet, View, Text} from 'react-native'
+import { StyleSheet, View, Text } from 'react-native'
```
Comment (lhmerino) [inline PopApp/Components/Attendee.js:2]: Seems there are import style discrepancies (spaces vs no spaces in imports)? (Discrepancy 1)

File: PopApp/Components/CameraButton.js [hunk 2]
```diff
@@ -1,21 +1,21 @@
 import React from 'react'
-import {StyleSheet, Image} from 'react-native'
-import { TouchableOpacity } from 'react-native-gesture-handler'
+import {StyleSheet, Image, View, TouchableOpacity} from 'react-native'
```
Comment (lhmerino) [inline PopApp/Components/CameraButton.js:2]: Discrepancy 1

**Review by lhmerino — CHANGES_REQUESTED** [review]
Looks great! Just a subtle style discrepancy with imports.

**Comment by RaulinN** [comment]
Fixed! Well spoted @lhmerino! We unified all our imports to reflect our [coding guidelines](https://thoughtbot.com/blog/structure-for-styling-in-react-native).

File: PopApp/App.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
-import 'react-native-gesture-handler'
```
Comment (imaximumGit) [inline PopApp/App.js:1]: The `import 'react-native-gesture-handler'` is compulsory to use react-navigation see [here](https://reactnavigation.org/docs/getting-started)

**Review by imaximumGit — CHANGES_REQUESTED** [review]
Just need a little modification to follow the instructions of react navigation

**Review by lhmerino — APPROVED** [review]

File: PopApp/App.js (hunk 3 above)
Comment (RaulinN) [reply PopApp/App.js]: Indeed my bad! Added it back.

I will remember next time to double check before deleting an import that IntelliJ deems useless! ^^'

**Review by imaximumGit — APPROVED** [review]

**Pull request merged**